birdhouse/components/canarie-api/default.env

```diff
@@ -1,7 +1,7 @@
 # Folder inside "proxy" container to drop extra monitoring config
 export CANARIE_MONITORING_EXTRA_CONF_DIR="/conf.d"
 
-export PROXY_IMAGE="pavics/canarieapi:0.7.1"
+export CANARIE_IMAGE="pavics/canarieapi:1.0.0"
```

Comment:
Use a `CANARIE_VERSION` and other `CANARIE_...` similar variables as for other components?